app/views/miq_request/pre_prov.html.haml

```diff
@@ -7,3 +7,4 @@
       :ajax_buttons                 => true,
       :continue_button              => true,
       :noreset                      => true})
+= javascript_pack_tag "manageiq-ui-classic/toolbar-actions-common"
```

Comment:
If your pack is named `*-common` you don't have to import it, it's imported automatically.